lib/chibi/json.c

```diff
@@ -80,8 +81,12 @@ sexp parse_json_string (sexp ctx, sexp self, sexp str, const char* s, int* i, co
         from = to+1;
         break;
       case 'u':
-        strncpy(utf_tmp, s+to+1, 5);
-        tmp = sexp_make_string(ctx, sexp_make_fixnum(1), sexp_make_character(strtoll(utf_tmp, NULL, 16)));
+        strncpy(utf_tmp, s+to+1, 4);
```

Comment:
Why the copy?  We can just use s.

Reply:
in the case of `\u12231243352462` strtol is going to take the whole number with no stop. We only need to parse the 4 chars and I didn't find a better way to do it.